packages/react-integration/demo-app-ts/src/components/demos/DropdownDemo/DropdownDemo.tsx

```diff
@@ -140,7 +140,7 @@ export class DropdownDemo extends React.Component<{}, DropdownState> {
           id="dropdown"
           onSelect={this.onSelect}
           toggle={
-            <DropdownToggle id="toggle-id" onToggle={this.onToggle} iconComponent={CaretDownIcon}>
```

Comment:
Can you also add the new icon prop to the demo.

Reply:
Updated this instance of DropdownToggle to include new icon prop